src/Profile.js

```diff
@@ -50,62 +50,19 @@ export default function CreatedGrid() {
     console.log(user.photoUrl);
   }
 
-  return (
-    <div>
-      <Card className={classes.root}>
-        <CardActionArea>
-          <CardMedia className={classes.media} />
-
-          <CardContent>
-            <Typography gutterBottom variant="h5" component="h2">
-              Gridlock
-            </Typography>
-            <Typography variant="body2" color="textSecondary" component="p">
-              Kousei course 2
-            </Typography>
-          </CardContent>
-        </CardActionArea>
-        <CardActions>
-          <Button size="small" color="primary">
-            Play
-          </Button>
-          <Button size="small" color="primary">
-            Delete
-          </Button>
-        </CardActions>
-      </Card>
-    </div>
-  );
-}
-
-// return (
-//   <div className="board">
-//     {grid_list.map((grid, i) => {
-//       return (
-//         <Card className={classes.root}>
-//           <CardActionArea>
-//             <CardMedia className={classes.media} />
-
-//             <CardContent>
-//               <Typography gutterBottom variant="h5" component="h2">
-//                 {grid.title}
-//               </Typography>
-//               <Typography variant="body2" color="textSecondary" component="p">
-//                 {username}
-//               </Typography>
-//             </CardContent>
-//           </CardActionArea>
-//           <CardActions>
-//             <Button size="small" color="primary">
-//               Play
-//             </Button>
-//             <Button size="small" color="primary">
-//               Delete
-//             </Button>
-//           </CardActions>
-//         </Card>
-//       );
-//     })}
-//   </div>
-// );
-// }
+    return (
+      <div className="board">
+        <img src={profilpic} alt="Girl in a jacket" width="150" height="150" />
+        {grid_list.map((grid, i) => {
```

Comment:
What is i?

Reply:
Not important, but I noticed it was never used. Resolving

Reply:
that was what swarnim sent into the hangout when he couldn't pull and push. I just used what is with master now